docs/contributors/dev_environment_setup.md

```diff
@@ -23,7 +23,7 @@ This guide will walk you through setting up your machine for contributing to Fir
 
 You will need a few prerequisites set up on your machine before you can build FireFly from source. We recommend doing development on macOS, Linux, or WSL 2.0.
 
-- [Go (1.17 or newer)](https://golang.org/dl/)
+- [Go (1.18 or newer)](https://golang.org/dl/)
```

Comment:
We've alway said "or newer" assuming that newer versions would be backward compatible, but in practice that hasn't worked out so well due to tooling and other frameworks. Do these changes also make it work with Go 1.19 (I can try it if you haven't), or should we just drop the "or newer" here?

Reply:
Okay, I got this branch to build with 1.18, but it is failing on 1.19 with:
```
INFO [runner] linters took 26.79923825s with stages: goanalysis_metalinter: 26.721642167s, structcheck: 13.959µs 
pkg/core/pin.go:40: File is not `gofmt`-ed with `-s` (gofmt)
//
pkg/dataexchange/plugin.go:52: File is not `gofmt`-ed with `-s` (gofmt)
//     - Example would be a logical filesystem path like "local/namespace/ID"
pkg/database/plugin.go:525: File is not `gofmt`-ed with `-s` (gofmt)
//
internal/events/websockets/websocket_connection.go:22:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
internal/shareddownload/operations.go:22:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
internal/spievents/websockets.go:22:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
```

Reply:
Resolved by #1097